README.md

```diff
@@ -29,8 +29,8 @@ The attributes listed below are used in *course.json* to configure **Adapt Graph
 >**\_onPauseRewind** (Boolean): Rewind when the pause button is clicked. Defaults to `false`.
 
 ----------------------------
-**Version number:**  0.0.17<br />
-**Framework versions:**  >=5.8<br />
+**Version number:**  0.1.0<br />
```

Comment:
@oliverfoster I believe we've been removing the version number from the README.md docs. That way, we don't have to keep it updated when we add the automated release stuff.